theme/themes/pastanaga/globals/site.overrides

```diff
@@ -1,3 +1,5 @@
 /*******************************
         Global Overrides
 *******************************/
+
+@import (multiple) '../extras/fonts';
```

Comment:
@giuliaghisini Shouldn't we import `fonts.less` in https://github.com/plone/volto/blob/staticize_font/theme/themes/pastanaga/extras/main.less#L560 ?

Reply:
@giuliaghisini I think @nileshgulia1 is correct, shouldn't we reference it from main.less?

Reply:
To put it in site.overrides was my idea in the first place, sorry 😝 
We "removed" if from site.variables, so it felt like the most appropriate place. In addition, it is more connected with how semantic-ui works. I was not a fan of importing from a different folder though, so I'm ok with changing this as you suggest.